client/components/push-notification/settings.jsx

```diff
@@ -0,0 +1,151 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Dialog from 'components/dialog';
+import Gridicon from 'components/gridicon';
+import Notice from 'components/notice';
+import observe from 'lib/mixins/data-observe';
+
+const debug = debugFactory( 'calypso:push-notification-settings' );
+
+export default  React.createClass( {
+	displayName: 'PushNotificationSettings',
+
+	mixins: [ observe( 'pushNotifications' ) ],
+
+	propTypes: {
+		pushNotifications: React.PropTypes.object
+	},
+
+	getInitialState: function() {
+		return {
+			showDialog: false
+		};
+	},
+
+	clickHandler: function() {
+		debug( 'Handling click for state %s', this.props.pushNotifications.state );
+
+		if ( 'subscribed' === this.props.pushNotifications.state ) {
+			this.props.pushNotifications.unsubscribe();
+		}
+
+		if ( 'unsubscribed' === this.props.pushNotifications.state ) {
+			this.props.pushNotifications.subscribe();
+		}
+	},
+
+	getBlockedInstruction: function() {
+		return (
+			<Dialog isVisible={ this.state.showDialog } className="push-notification-instruction__dialog" onClose={ this._onCloseDialog }>
+				<div className="push-notification-instruction__content">
+					<div>
+						<div className="push-notification-instruction__title">{ this.translate( 'Enable Browser Notifications' ) }</div>
+						<div className="push-notification-instruction__step">
+							<img height="180px" width="180px" src="/calypso/images/push-notifications/address-bar.svg" />
```

Comment:
We are trying to inline all svgs in Calypso instead of doing extra requests. Would that be possible here?